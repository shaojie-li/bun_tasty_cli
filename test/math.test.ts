import { expect, test } from "bun:test";

test("2 + 2", () => {
  expect(2 + 2).toBe(4);
});

test("snapshot", async () => {
  expect({ c: new Date() }).toMatchSnapshot({ c: expect.any(Date) });
}); 