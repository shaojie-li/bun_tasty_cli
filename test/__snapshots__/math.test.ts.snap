// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "c": Any<Date>,
}
`;
